./exec_getRFMeasError_v_1_0_2.py